tools/cabana/dbc/generate_dbc_json.py

```diff
@@ -2,12 +2,13 @@
 import argparse
 import json
 
+from opendbc.car import Bus
 from opendbc.car.fingerprints import MIGRATION
 from opendbc.car.values import PLATFORMS
 
 
 def generate_dbc_json() -> str:
-  dbc_map = {platform.name: platform.config.dbc_dict['pt'] for platform in PLATFORMS.values() if platform != "MOCK"}
+  dbc_map = {platform.name: platform.config.dbc_dict[Bus.pt] for platform in PLATFORMS.values() if platform != "MOCK"}
```

Comment:
@robbederks please make sure to check for references in the upstream repos when making large refactors like this. If in openpilot, then xx, etc.